lib/reverse_words.rb

```diff
@@ -20,7 +20,6 @@ def reverse_words(my_words)
       end
 
       my_words[i..k] = reverse_word(my_words[i..k])
```

Comment:
As we saw in class, in the first few slides of https://drive.google.com/file/d/0B__DV26QHsH4bWJmS1A0QXBad1U/view?usp=sharing, the substring method e.g. `my_words[i..k]` is creating a new string object. This will increase your space complexity and not manipulate your string in-place. To avoid this, have your reverse_word method take as input parameter, my_words as well as i and k as starting index and ending index. Then update your reverse_word method to only reverse characters from starting index to the ending index.